Council_Roles.md

```diff
@@ -2,53 +2,44 @@
 
 ### President
 
-1. Oversee all UofT Scientific Coders activities and be responsible for the internal actions of the council;
+1. Oversee all UofT Coders activities and be responsible for the internal actions of the council;
 2. Organize and preside over monthly council meetings;
-3. Ensure that UofT Scientific Coders affairs are conducted in a fiscally responsible manner;
-4. Marketing
+3. Ensure that UofT Coders affairs are conducted in a fiscally responsible manner;
+4. Marketing:
   * Send out recruitment emails to various avenues to increase awareness
 5. Make final decisions in the event where decisions are split;
 
 ### Treasurer
 
-1.  Act as UofT Coders financial officer, and together with the
-President, ensure the groups financial stability;
+1.  Act as UofT Coders financial officer, and together with the President, ensure the groups financial stability;
 2. Oversee the group bank account;
-3. Keep receipts for all financial transactions, and record on the 
-[council repository](https://github.com/UofTCoders/council/tree/master/treasurer) 
-all debits, credits, and cheques released, as well as the
-dates and remaining bank balance for
-maintaining and organizing bank records, and for
-conducting annual audits (if applicable) in accordance with
-policies established by Student Affairs, University of
-Toronto;
+3. Keep receipts for all financial transactions, and record on the [council repository](https://github.com/UofTCoders/council/tree/master/treasurer) all debits, credits, and cheques released, as well as the dates and remaining bank balance for maintaining and organizing bank records, and for conducting annual audits (if applicable) in accordance with policies established by Student Affairs, University of Toronto;
 4. Apply for funding and sponsorship for the group, as applicable;
 5. Prepare an annual budget at the start of fiscal year;
-6. Present current statements of accounts versus budget,
-and cash flow positions at Executive Council meetings;
-7. Prepare of an annual financial report at the end of the
-fiscal year.
+6. Present current statements of accounts versus budget, and cash flow positions at Executive Council meetings;
```

Comment:
Hmm, good point... given that there is a README always available, number 7. doesn't make sense... However, number 6. is really important. I think we should do this soon, maybe at the next meeting (not tomorrow).